types/src/transaction/authenticator.rs

```diff
@@ -358,6 +358,9 @@ impl TransactionAuthenticator {
                 AccountAuthenticator::MultiKey { authenticator } => {
                     single_key_authenticators.extend(authenticator.to_single_key_authenticators()?);
                 },
+                AccountAuthenticator::NoAccountAuthenticator => {
+                    //  TODO: check this.
```

Comment:
No authenticator so there's nothing to do here?

Reply:
Yes, no authentication is added in this case. I updated the comment accordingly. (cc: @heliuchuan )